p2p-media.js

```diff
@@ -1,28 +1,43 @@
-import { supportsP2P, resolveP2PUrl } from './db.js'
+import { supportsP2P, resolveP2PUrl, isP2P } from './db.js'
 
 class P2PImage extends HTMLElement {
   constructor () {
     super()
+    this.attachShadow({ mode: 'open' })
 
+    // Create an img element
     this.img = document.createElement('img')
-    this.attachShadow({ mode: 'open' })
-    this.shadowRoot.appendChild(this.img)
+
+    // Load the CSS file
+    const style = document.createElement('style')
```

Comment:
Couldn't we just add a regular style tag with `@mport` in the template?

TBH I think we should avoid the shadow root entirely and just embed the image into the plain dom and use regular css targetting the custom element.

Reply:
Same for the video tag